samples/sample_decode/src/pipeline_decode.cpp

```diff
@@ -1034,6 +1034,11 @@ mfxStatus CDecodingPipeline::InitVppParams()
         (  m_mfxVppVideoParams.vpp.Out.FourCC == MFX_FOURCC_P010
 #if (MFX_VERSION >= 1027)
         || m_mfxVppVideoParams.vpp.Out.FourCC == MFX_FOURCC_Y210
+#endif
+#if (MFX_VERSION >= 1330)
```

Comment:
1330?

Reply:
corrected to 1331

Reply:
No. https://github.com/Intel-Media-SDK/MediaSDK/blob/c8ab21caa250934b456a744d8dd90ae75285f9c9/api/include/mfxstructures.h#L107-L109

Reply:
corrected to 1031